testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/HibernateNativeAPITransactionTestCase.java

```diff
@@ -79,6 +80,9 @@ public class HibernateNativeAPITransactionTestCase {
     public static void beforeClass() throws NamingException {
         // TODO This can be re-looked at once HHH-13188 is resolved. This may require further changes in Hibernate.
         AssumeTestGroupUtil.assumeSecurityManagerDisabled();
+        // This test needs to be recompiled against Hibernate ORM 6 (WFLY-16178) in order to pass.
+        assumeTrue(System.getProperty("ts.ee9") == null);
```

Comment:
You need to check for ts.bootable.ee9 as well as that enables the profile used in jobs like https://ci.wildfly.org/viewType.html?buildTypeId=WF_MainNightlyJobs_WildFlyPreviewBootableJarLinuxJdk11